SU-SWS/decanter
# removed FA from project; updagraded to latest normalize

# READY FOR REVIEW

# Summary
- Removed FontAwesome from project entirely
- Upgraded to latest version of normalize

# Needed By (Date)
- N/A

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Ensure FA is gone
3. Ensure new normalize is added (8.0.0)

# Affected Projects or Products
- Decanter
- Stanford Basic

# Associated Issues and/or People
- https://github.com/SU-SWS/decanter/issues/99
- https://github.com/SU-SWS/decanter/issues/102

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
